tests/testsuite/artifact_dep.rs

```diff
@@ -1267,17 +1344,15 @@ fn no_cross_doctests_works_with_artifacts() {
     p.cargo("test -Z bindeps --target")
         .arg(&target)
         .masquerade_as_nightly_cargo(&["bindeps"])
-        .with_stderr(&format!(
-            "\
+        .with_stderr_data(str![[r#"
 [LOCKING] 2 packages to latest compatible versions
-[COMPILING] bar v0.5.0 ([CWD]/bar)
-[COMPILING] foo v0.0.1 ([CWD])
-[FINISHED] `test` profile [unoptimized + debuginfo] target(s) in [..]
-[RUNNING] [..] (target/{triple}/debug/deps/foo-[..][EXE])
+[COMPILING] bar v0.5.0 ([ROOT]/foo/bar)
+[COMPILING] foo v0.0.1 ([ROOT]/foo)
+[FINISHED] `test` profile [unoptimized + debuginfo] target(s) in [ELAPSED]s
+[RUNNING] unittests src/lib.rs (target/[TARGET]/debug/deps/foo-d7d4d3e3948ead58)
```

Comment:
Is that hash going to cause a problem?

Reply:
Thanks for catching this, I thought I caught all of them